python/tests/test_util.py

```diff
@@ -540,3 +540,27 @@ def test_length_bad_value(self, length):
     def test_length_bad_type(self, length):
         with pytest.raises(TypeError, match="argument must be a string"):
             tskit.random_nucleotides(length)
+
+
+class TestAlleleRemap:
+    a = ["A", "C", "G", "T"]
+    b = ["G", "C"]
+    c = []
+
+    def test_a_bigger_than_b(self):
```

Comment:
Maybe rename these to `test_from_bigger_than_to` and `test_to_bigger_than_from` as currently `test_a_smaller_than_b` looks like an error as it isn't.